iroh-gossip/Cargo.toml

```diff
@@ -34,10 +34,11 @@ once_cell = "1.18.0"
 genawaiter = { version = "0.99.1", default-features = false, features = ["futures03"] }
 
 [dev-dependencies]
-tracing-subscriber = { version = "0.3", features = ["env-filter"] }
 clap = { version = "4", features = ["derive"] }
-url = "2.4.0"
+iroh-test = { path = "../iroh-test" }
```

Comment:
these should have the current version as well, makes it easier when I publish things